### PR TITLE
Implement qPlatform_shell

### PR DESCRIPTION
**Pull request by fjricci**
This was mostly implemented already as the now-non-existent qPlatform_RunCommand packet.

### CODE REVIEW

File: Sources/Host/POSIX/ProcessSpawner.cpp [hunk 1]
```diff
@@ -89,6 +89,23 @@ bool ProcessSpawner::setExecutable(std::string const &path) {
     return false;
 
   _executablePath = path;
+  _shell = false;
+  return true;
+}
+
+bool ProcessSpawner::setShellCommand(std::string const &command) {
+  if (_pid != 0)
```
Comment (a20012251) [inline Sources/Host/POSIX/ProcessSpawner.cpp]: { }

File: Sources/Host/Windows/ProcessSpawner.cpp [hunk 2]
```diff
@@ -34,6 +34,14 @@ bool ProcessSpawner::setExecutable(std::string const &path) {
   return true;
 }
 
+bool ProcessSpawner::setShellCommand(std::string const &command) {
+  if (_pid != 0)
```
Comment (a20012251) [inline Sources/Host/Windows/ProcessSpawner.cpp]: {}

**Comment by a20012251** [comment]
@facebook-github-bot shipit
just add the  { }

File: Headers/DebugServer2/Host/POSIX/ProcessSpawner.h [hunk 3]
```diff
@@ -53,13 +53,15 @@ class ProcessSpawner {
   int _exitStatus;
   int _signalCode;
   ProcessId _pid;
+  bool _shell;
```
Comment (sas) [inline Headers/DebugServer2/Host/POSIX/ProcessSpawner.h]: You should also initialize `_shell` in the constructor.

**Comment by sas** [comment]
@facebook-github-bot shipit

You should just add the `{}` @a20012251 mentioned and add an initializer for `_shell` in `POSIX::ProcessSpawner`. Then I think this is good to go.

**Pull request merged**